session7/src/com/itschool/course/Homework2.java

```diff
@@ -0,0 +1,13 @@
+package com.itschool.course;
+
+public class Homework2 {
+    //EX 3.
```

Comment:
again, this refers to Ex3 from session6, but this class is in module session7. Also the package name is `com.itschool.course` and it should have been `com.itschool.homework`.
However, the exercise is correctly solved! :+1: